org.eclipse.scanning.test/src/org/eclipse/scanning/test/command/MScanTest.java

```diff
@@ -13,48 +13,48 @@ public MScanTest() {
 	}
 
 	@Test
-	public void testGridScan() throws Exception {
+	public void testGridScan() {
 		pi.exec("mscan(grid(axes=('xNex', 'yNex'), start=(0, 0), stop=(10, 10), count=(2, 2), snake=True), det=detector('mandelbrot', 0.001))");
 	}
-		
+
 	@Test
-	public void testGridScanNoDetector() throws Exception {
+	public void testGridScanNoDetector() {
 		pi.exec("mscan(grid(axes=('xNex', 'yNex'), start=(0, 0), stop=(10, 10), count=(2, 2), snake=True))");
 	}
-	
+
 	@Test
-	public void testStepAroundGridScanNoDetector() throws Exception {
+	public void testStepAroundGridScanNoDetector() {
 		pi.exec("mscan([step(axis='energy', start=300, stop=310, step=5), grid(axes=('xNex', 'yNex'), start=(0, 0), stop=(10, 10), count=(2, 2), snake=True)])");
 	}
-	
+
 	@Test
-	public void testMultiStep() throws Exception {
+	public void testMultiStep() {
 		pi.exec("mscan(mstep(axis='energy', stepModels=[StepModel('energy', 300, 310, 5)]))");
 	}
-	
+
 	@Test
-	public void testMulti1StepAroundGridScanNoDetector() throws Exception {
+	public void testMulti1StepAroundGridScanNoDetector() {
 		pi.exec("mscan([mstep(axis='energy', stepModels=[StepModel('energy', 300, 310, 5)]), grid(axes=('xNex', 'yNex'), start=(0, 0), stop=(10, 10), count=(2, 2), snake=True)])");
 	}
-	
+
 	@Test
-	public void testMulti2StepAroundGridScanNoDetector() throws Exception {
+	public void testMulti2StepAroundGridScanNoDetector() {
 		pi.exec("mscan([mstep(axis='energy', stepModels=[StepModel('energy', 290, 300, 5), StepModel('energy', 300, 310, 5)]), grid(axes=('xNex', 'yNex'), start=(0, 0), stop=(10, 10), count=(2, 2), snake=True)])");
 	}
-	
+
 	@Test
-	public void testGridWithROIScan() throws Exception {
+	public void testGridWithROIScan() {
 		pi.exec("mscan(grid(axes=('xNex', 'yNex'), start=(0.0, 1.0), stop=(10.0, 12.0), count=(3, 2), snake=False, roi=[circ(origin=(0.0, 1.0), radius=2.0)]), det=detector('mandelbrot', 0.001))");
 	}
-		
+
 	@Test
-	public void testGridScanWithGoodTimeout() throws Exception {
+	public void testGridScanWithGoodTimeout() {
 		pi.exec("mscan(grid(axes=('xNex', 'yNex'), start=(0, 0), stop=(10, 10), count=(2, 2), snake=True), det=detector('mandelbrot', 1.2, timeout=2))");
 	}
 
 	@Test
-	public void testI15_1Case() throws Exception {
-		pi.exec("mscan(path=[grid(axes=('stage_x', 'stage_y'), start=(-1.5, -1.0), stop=(0.5, 1.0), count=(2, 2), snake=False)], mon=['s1MockNeXusSlit'], det=[detector('mandelbrot', 0.001, maxIterations=500, escapeRadius=10.0, columns=301, rows=241, points=100, maxRealCoordinate=1.5, maxImaginaryCoordinate=1.2, realAxisName='stage_x', imaginaryAxisName='stage_y', enableNoise=False, noiseFreeExposureTime=5.0, saveImage=True, saveSpectrum=True, saveValue=True)])");
+	public void testI15_1Case() {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)